_posts/2018-12-18-downloader.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Initial release of OMERO.downloader
+categories: tools
+---
+
+# Introducing OMERO.downloader
+
+For OMERO to properly fulfill the role of being a useful repository for
+microscopy images its users must have easy access to that data. As data
+sets grow in size it becomes a correspondingly greater challenge to
+provide access to that data. This motivates the creation of server-side
+solutions such as the IDR's
+[Virtual Analysis Environment](https://idr-analysis.openmicroscopy.org/).
+
+For the past couple of years the OME team has been investigating ways to
+improve users' ability to obtain data from OMERO for client-side storage
+and processing. We now release OMERO.downloader v0.1.0 which is still an
```

Comment:
I would talk first about what downloader _is_ rather than what it's isn't. i.e. "We now release a preview version of ... " then the next paragraph, then the "missing many features" bit.